app/models/metais/project_document.rb

```diff
@@ -0,0 +1,8 @@
+class Metais::ProjectDocument < ApplicationRecord
+  self.table_name = 'metais.dokumenty_projekty'
```

Comment:
Nazvy tabuliek by som asi drzal v rails konvencii po anglicky. Viem ze to je divne ale myslim ze vsade v ekosysteme mame tabulky english aj ked niektore atributy su slovenske.

Reply:
Aha, no okay. Ja som pozrel ITMS, čo je veľká časť harvestera a tam boli po slovensky, tak som dal rovnako. Ok, premenujem.